src/docs/testing/index.md

```diff
@@ -41,19 +40,18 @@ different kinds of tests:
 |----------------------|--------|--------|-------------|
 | **Confidence**       | Low    | Higher | Highest     |
 | **Maintenance cost** | Low    | Higher | Highest     |
-| **Dependencies**     | Few    | More   | Lots        |
+| **Dependencies**     | Few    | More   | Most        |
 | **Execution speed**  | Quick  | Slower | Slowest     |
 {:.table.table-striped}
 
-**Tip**: As a rule of thumb a well-tested app has a very high number of unit
-and widget tests, tracked by [code coverage](https://en.wikipedia.org/wiki/Code_coverage),
-and a good number of integration tests covering all the important usage
-scenarios.
+**Tip**: A well-tested app has many unit and widget tests,
```

Comment:
Nit: Could you wrap this tip text in `{{site.alert.tip}}` ... `{{site.alert.end}}`? (Search the site files for examples.)